asapdiscovery-data/asapdiscovery/data/scripts/sdf_to_smiles_list.py

```diff
@@ -0,0 +1,36 @@
+"""
+Load in a multi-ligand SDF file and write out a txt file with the format `SMILES <space> ligand_name`
+Ligand name expected to be in the Compound_ID field of the sdf file.
+"""
+from argparse import ArgumentParser
+
+
+def get_args():
+    parser = ArgumentParser(description="")
+    parser.add_argument(
+        "-s",
+        "--sdf_fn",
+        required=True,
+        help="SDF file containing molecules of interest.",
+    )
+    parser.add_argument("-o", "--output_fn", required=True, help="Output file.")
+    return parser.parse_args()
+
+
+def main():
+    args = get_args()
+    from asapdiscovery.data.openeye import load_openeye_sdfs, oechem
+
+    mols = load_openeye_sdfs(args.sdf_fn)
+    output_lines = [
+        f"{oechem.OEGetSDData(mol, 'SMILES')} {oechem.OEGetSDData(mol, 'Compound_ID').rstrip()}\n"
+        for mol in mols
+    ]
```

Comment:
might be good to have a fallback in case either of the fields aren't stored. eg check with `oechem.OEHasSDData`, and if it's not present, you can use `oechem.OEMolToSmiles` for the SMILES and maybe just the compound number or something for compound ID

Reply:
done https://github.com/choderalab/asapdiscovery/pull/105/commits/0df4dd6831b443ddf0b1ec6c37eb24cf8fc7085d